docs/setup/installation.md

```diff
@@ -18,8 +20,127 @@ If you want to build SourceCred from source (for instance to get bleeding edge f
 If you don't have it, install [yarn](https://classic.yarnpkg.com/lang/en/). Then run yarn to install SourceCred and its dependencies.
 
 `yarn` 
+<<<<<<< HEAD
```

Comment:
Seems like there's an unresolved merge conflict here. Did you mean to edit this page?